deploy/efs-csi/05-scc.yaml

```diff
@@ -0,0 +1,39 @@
+# A clone of the `anyuid` scc allowing dedicated-admins to create pods
```

Comment:
This is concerning, as this may allow root pods.

Reply:
That's its purpose. NFS mounts are created with root ownership, so out of the box they would be unusable to the customer. We could go fix it for them, but again that puts us in a middleman role. So instead we open this up for them and show them how to create a temporary root pod to change the permissions, and then they promise not to do it again.

Reply:
@cblecker this is an accepted risk, the customer will also be provided with an enhanced dedicated-admin. This is to remove SRE's involvement from the process.